.github/renovate.json5

```diff
@@ -2,4 +2,5 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>canonical/data-platform//renovate_presets/charm.json5"],
   "reviewers": ["team:data-platform-postgresql"],
+  "baseBranches": ["/^*\\/edge$/"],
```

Comment:
Match end of the branch name as well, so we don't update things like `14/edge-test`.